tests/UI/specs/ViewDataTable_spec.js

```diff
@@ -196,10 +196,21 @@ describe("ViewDataTableTest", function () { // TODO: should remove Test suffix f
         await page.click('.activateExportSelection');
         await page.waitFor('#reportExport .btn');
 
-        dialog = await page.$('.ui-dialog');
+        var dialog = await page.$('.ui-dialog');
```

Comment:
just a note, but the UI tests are run in puppeteer so you can use es6+ in them.

Reply:
@diosmosis I modified to `let`